src/app/components/OptimizelyPageMetrics/index.test.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { RequestContext } from '#contexts/RequestContext';
+import OptimizelyPageMetrics from '.';
+
+jest.mock('./PageCompleteTracking', () => () => (
+  <div data-testid="page-complete-tracking" />
+));
+jest.mock('./ScrollDepthTracking', () => () => (
+  <div data-testid="scroll-depth-tracking" />
+));
+jest.mock('./PageViewTracking', () => () => (
+  <div data-testid="page-view-tracking" />
+));
+
+const renderWithContext = (ui, { isAmp = false } = {}) => {
+  return render(
+    <RequestContext.Provider value={{ isAmp }}>{ui}</RequestContext.Provider>,
+  );
+};
+
+describe('OptimizelyPageMetrics', () => {
+  it('returns null when isAmp is true', () => {
+    const { container } = renderWithContext(
+      <OptimizelyPageMetrics trackPageView trackPageDepth trackPageComplete />,
+      { isAmp: true },
+    );
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('renders no tracking components by default when all tracking flags are false', () => {
+    renderWithContext(<OptimizelyPageMetrics />, { isAmp: false });
```

Comment:
You don't technically need to set isAmp = false in any of the tests, as it is defaulted above
```suggestion
    renderWithContext(<OptimizelyPageMetrics />);
```

Reply:
Blame me hastily C+V from AI 🤖

Reply:
Sorry my bad! Missed this one before